website/pages/docs/internals/telemetry.mdx

```diff
@@ -86,7 +86,7 @@ These metrics represent operational aspects of the running Vault instance.
 | `vault.core.handle_request`          | Duration of time taken by requests handled by Vault core                                                                                                                                            | ms   | summary |
 | `vault.core.handle_login_request`    | Duration of time taken by login requests handled by Vault core                                                                                                                                      | ms   | summary |
 | `vault.core.leadership_setup_failed` | Duration of time taken by cluster leadership setup failures which have occurred in a highly available Vault cluster. This should be monitored and alerted on for overall cluster leadership status. | ms   | summary |
-| `vault.core.leadership_lost`         | Duration of time taken by cluster leadership losses which have occurred in a highly available Vault cluster. This should be monitored and alerted on for overall cluster leadership status.         | ms   | summary |
+| `vault.core.leadership_lost`         | Duration of time taken by cluster leadership losses which have occurred in a highly available Vault cluster. This should be monitored and alerted on for overall cluster leadership status. Any count greater than zero means that Vault experienced a leadership change and could potentially be cause for alerting.  Do note that a high mean value here is considered better than a low value, because a low value means there is leadership flapping. The measured value of this metric answers the question "how long was this server the leader, when it lost leadership?" | ms   | summary |
```

Comment:
```suggestion
| `vault.core.leadership_lost`         | Measures the amount of time that the HA cluster leader was the leader, at the point when it lost leadership. This should be monitored and alerted on for overall cluster leadership status. Any count greater than zero means that Vault experienced a leadership change and could potentially be cause for alerting.  Low values are a signal of leadership flapping. | ms   | summary |
```

Reply:
It's really the first sentence that I feel needs editing, "Duration of time taken by cluster leadership losses" doesn't seem to mean anything.